media/js/base/mozilla-fxa-form.js

```diff
@@ -81,6 +80,7 @@ Mozilla.FxaForm = (function(Mozilla) {
             // This allows non-Firefoxes to create accounts.
             fxaFormContextField.remove();
```

Comment:
This is my mistake. `ChildNode.remove()` isn't supported in IE, so these fields aren't being omitted and FxA tries to sign a non-Firefox into Sync, which is impossible so it just spins and spins. https://caniuse.com/#feat=childnode-remove

Alas, removing them still doesn't make the form work in IE, it just throws a 500 error, but that's arguably better than a loading animation that never finishes.

Reply:
As a fix that doesn't on `remove()`, we could just set the fields to `disabled` ?

Reply:
The changes I've added here don't try to address IE, just fix the metrics being missing from non-FF browsers. Let's try and fix IE in a followup?

Reply:
Oh wait, nevermind it took me a minute to understand the problem. I'll take a look